app/src/main/java/com/cornellappdev/score/components/SportCard.kt

```diff
@@ -47,16 +57,17 @@ fun SportCard(
         colors = CardDefaults.cardColors(containerColor = Color.White),
         modifier = modifier
             .clip(cardShape)
+            .shadow(elevation = 6.dp, spotColor = SpotColor, ambientColor = AmbientColor)
             .then(
                 if (topCornerRound) {
-                    Modifier.border(.4.dp, Color.Gray, cardShape)
+                    Modifier
+                        .border(width = 1.dp, color = Color(0xFFEAEAEA),cardShape)
```

Comment:
What's this color? I see it appearing a few times in this file.

If it's reused a lot and has a name, please pull this out into `Color.kt` as a new constant.